documentation/airport-format.md

```diff
@@ -1,309 +1,551 @@
-          "28L": [["SENZY", "A25+"], "ZUPAX"],  // at that fix, separated by a pipe symbol ('|'). For example, see the following: ["FIXNAME", "A50-|S220+"]. In
-          "28R": [["SENZY", "A25+"], "ZUPAX"]   // that example, restrictions of Altitude 5,000' or lower, and Speed 220kts or higher would be placed on that fix.
+],
+```
+- **name** - Name of each runway in the pair.  Names should reflect a 180 degree difference. so if one runway is `09` the other runway should be `270`.
+- **name_offset** -
+- **end** - Latitude, Longitude, and Elevation of the runway threshold (the spot where the numbers would be painted). _see [lat, lon, elev](#latitude-longitude-elevation) for formatting_
+- **ils** - Boolean property used to indicate if a runway is ILS capable
+
+Runways are defined in pairs because a runway can be used from either direction.  This makes defining runways a little tricky, so special attention should be paid to how the data is set up.  For each property, the first value will be considered part of the first runway and the second property for the second runway.  If you were to take the above example and extract each runway's properties, you would end up with the following two objects:
+
+```javascript
+// Runway 07L
+{
+    "name": "07L",
+    "name_offset": [0, 0],
+    "end": [
+        ["N36d4m34.82", "W115d10m16.98", "2179ft"]
+
+    ],
+    "ils": false
+}
+
+// Runway 07R
+{
+    "name": "25R",
+    "name_offset": [0, 0],
+    "end": [
+        ["N36d4m35.05", "W115d7m15.93", "2033ft"]
+    ],
+    "ils": true
+}
+```
+
+## Standard Procedures
+
+Standard Procedures consist of SIDs and STARs and, at a very high level, all contain three segments:
+1. Entry - the start of the procedure. can be on one of (possibly) several transition routes that feed into a central segment (the Body)
+2. Body - shared segment that all aircraft on the route will follow
+3. Exit - end of the procedure. can be one of (possibly) several exit segments
+
+This structure is used to work with both SIDs and STARs within the app.  Though it's not important to know for an airport file, it is a good thing to keep in mind.
+
+Fixes within the segments can be defined in several different ways:
+```javascript
+// fix name only
+"07L": ["WASTE", "COMPS"]
+
+// fix name with altitude restriction
+"07L": ["WASTE", ["BAKRR", "A70"], "COMPS"]
+
+// fix name with speed restriction
+"07L": ["WASTE", ["BAKRR", "S200"], "COMPS"]
+
+// fix name with altitude and speed restriction with min or max
+"07L": ["WASTE", ["BAKRR", "A70+|S250-"], "COMPS"]
+```
+These definitions can be used within any `Entry`, `Body` or `Exit` segment of a standardRoute.
+
+
+### sids
+_All properties in this section are required for each route definition_
+
+```javascript
+"sids": {
+    "COWBY6": {
+        "icao": "COWBY6",
+        "name": "Cowboy Six",
+        "suffix": {"1L":"", "1R":"", "28L":"", "28R":""},
```

Comment:
Some countries use a number and letter "suffix" to represent the procedure when it ends at a different runway. For example, the BATEL arrival into EDDT is the `BATEL 6Z` into runways 8L/8R, and `BATEL 3H` into runways 26L/26R.

See [the arrival charts](http://nav.vatsim-germany.org/files/edww/charts/eddt/public/EDDT_STAR.pdf) for more context.